Update for PureScript 0.14

thomashoneyman opened the pull request.

thomashoneyman committed:
Update for PureScript 0.14

thomashoneyman commented on src/Halogen/VDom/StringRenderer.purs:
```diff
@@ -47,7 +47,7 @@ render getTagType renderAttrs renderWidget = go
       as' = maybe as (\(Namespace ns) -> "xmlns=\"" <> escape ns <> "\"" <> if S.null as then "" else " " <> as) maybeNamespace
     in
-      "<" <> name <> (if S.null as then "" else " ") <> as <>
+      "<" <> name <> (if S.null as then "" else " ") <> as' <>
```
`as'` was previously unused -- presumably it was supposed to be used here?

garyb commented:
Changes look good to me! Should we switch this to GH actions too? Especially since Travis seems to have done something weird with these builds.

krisajenkins committed:
Bugfix for the rendering of tags that have a namespace but not other attributes. (#11)

Due to a mixup with primed and non-primed variable names (`as` and
`as'`), if you have an element with a namespace but no attributes,
such as `SVG.g [] [ ... ]` it was rendering as: `<gxmlns="..."`.

This commit fixes that behaviour.

Co-authored-by: Kris Jenkins <kjenkins@confluent.io>
